ipatests/test_xmlrpc/test_add_remove_cert_cmd.py

```diff
@@ -11,6 +11,30 @@
 from ipatests.util import assert_deepequal, raises
 from ipatests.test_xmlrpc.xmlrpc_test import XMLRPC_test
 from ipatests.test_xmlrpc.testcert import get_testcert
+from ipatests.test_xmlrpc.tracker.user_plugin import UserTracker
+from ipatests.test_xmlrpc.tracker.idview_plugin import IdviewTracker
```

Comment:
This commit precludes the implementation of IdviewTracker. Please rebase them in the right order.